ansible_collections/f5networks/f5_modules/plugins/modules/bigip_virtual_server.py

```diff
@@ -3676,6 +3702,10 @@ def __init__(self):
                 ]
             ),
             mirror=dict(type='bool'),
+            auto_last_hop=dict(
+                default='default',
```

Comment:
you might wanna remove default, if a user modifies a  VS which has this value set to something else, and the user does not specfiy it in their play, the module will override it on the device